CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.2.3]
```

Comment:
if it is intended to update the version, please make sure to immediately implement it in AP. otherwise, revert version in version.json to 0.2.2 and mark this as Unreleased